lunatrace/bsl/frontend/src/pages/vulnerabilities/detail/DetailBody.tsx

```diff
@@ -87,6 +89,31 @@ export const VulnerabilityDetailBody: React.FunctionComponent<VulnerabilityDetai
                 ) : (
                   <span>No CVSS score</span>
                 )}
+                <hr />
+                <h4 className="">
+                  Your Projects Vulnerable: <span className="lighter">{vuln.findings.length}</span>{' '}
+                </h4>
+
+                {vuln.findings.map((f) => {
+                  const projectName = f.default_branch_build?.project?.name;
```

Comment:
if this isn't truthy, what happens? Does the displayed URL get busted up and not work? Should we be running a "filter" pass first?